src/services/user/services/AdminUsers.js

```diff
@@ -309,6 +326,17 @@ const formatBirthdayOfUsers = ({ result: { data: users } }) => {
 		}
 	});
 };
+const markUserAsDeletedHooks = () => ({
+	before: {
+		all: [authenticate('jwt')],
+		find: [disallow()],
+		get: [disallow()],
+		create: [disallow()],
+		update: [authenticate('jwt'), iff(isProvider('external'), isSuperHero())],
+		patch: [disallow()],
+		remove: [disallow()],
```

Comment:
please use remove instead of update. And you don't need jwt check in update if you have it already in all.

Reply:
done ;)